### PR TITLE
Added PR curve page (not linked yet)

### PR DESCRIPTION
**Pull request by y27choi**
### Linked issue(s):
Fixes KOL-2635

### What change does this PR introduce and why?
Added a page for PR curve (diagrams with proper style / dark mode support will be added as a separate PR)

### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

File: docs/metrics/pr-curve.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+Let's consider a simple binary classification example and plot a PR curve at a uniformly spaced range of thresholds.
+The table below shows six samples (four positive and two negative) sorted by their confidence score. Each inference
+is evaluated at each threshold: 0.25, 0.5, and 0.75. It's a negative prediction if its confidence score is below the
+evaluating threshold; otherwise, it's positive.
+
+<center>
+
+| Sample | <nobr>Confidence ↓</nobr> | Inference @ 0.25 | Inference @ 0.5 | Inference @ 0.75 |
+| --- | --- | --- | --- | --- |
+| Positive | 0.9 | <span style="color: green">Positive</span> | <span style="color: green">Positive</span> | <span style="color: green">Positive</span> |
```
Comment (y27choi) [inline docs/metrics/pr-curve.md]: Couldn't figure out ways to change font color based on the dark/light mode. If anyone has a good idea, let me know!

File: docs/metrics/pr-curve.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+<center>
+![pr.png](../assets/images/metrics-prcurve-example.png)
+</center>
+
+### Multiclass
+
+For multiple classes, micro or macro precision and recall can be used instead. It is also useful to plot a curve **per
+class** by binarizing the input so that we only consider the class of interest. This technique is known as
+**one-vs-rest** (OvR). With this strategy, we can have `n` PR curves for `n` unique classes.
```
Comment (munkyshi) [inline docs/metrics/pr-curve.md]: This feels a bit lacking in comparison to the `Example: Binary Classification` section -- feels like I'm left with more questions than answers (What is binarizing? What happens with OvR? Is there an example?)

File: docs/metrics/pr-curve.md (hunk 1 above)
Comment (munkyshi) [reply docs/metrics/pr-curve.md]: Here's an example (with bad color choices): https://github.com/kolenaIO/kolena/pull/154

**Review by munkyshi — COMMENTED** [review]
Would like to see multiclass and OvR a bit more fleshed out, but otherwise looks good.

File: docs/metrics/pr-curve.md (hunk 2 above)
Comment (y27choi) [reply docs/metrics/pr-curve.md]: Added an example for Multiclass

File: docs/metrics/pr-curve.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+![pr.png](../assets/images/metrics-prcurve-example-multiclass.png)
+</center>
+
+
+## Area Under the PR Curve (AUPRC)
+
+The area under the PR curve (AUPRC) is a **threshold-independent** metric that summarizes the performance of a model
+depicted by a PR curve. The greater the area, the better a model performs. The
+[average precision](./average-precision.md) is one particular method for calculating the AUPRC. With PR curves, we
+don’t have to go into any math and can visually conclude which curves indicate that a certain class or model has a
```
Comment (munkyshi) [inline docs/metrics/pr-curve.md]: nit: `don't have to go into any math` feels extraneous -- `we can visually conclude...` is more pithy.

**Review by munkyshi — APPROVED** [review]
One nit, but otherwise looks good!

**Comment by y27choi** [comment]
@pvanburen could you please take a look at these diagrams once you are done with the average precision ones?

**Pull request merged**